Assets/MixedRealityToolkit/InputSystem/Gaze/GazeProvider.cs

```diff
@@ -60,6 +64,15 @@ public class GazeProvider : MonoBehaviour, IMixedRealityGazeProvider
         [Tooltip("Transform that should be used to represent the gaze position and rotation. Defaults to CameraCache.Main")]
         private Transform gazeTransform = null;
 
+        [SerializeField]
+        [Range(0.1f, 5f)]
+        [Tooltip("Minimum head velocity threshold")]
+        private float minHeadVelocityThresh = 0.5f;
+
+        [SerializeField]
+        [Tooltip("Maximum head velocity threshold")]
+        private float maxHeadVelocityThresh = 2f;
```

Comment:
maxHeadVelocityThresh needs a range as well. Also need to verify that max >= min.

Reply:
any suggestion on what these ranges should be?